src/Logic.hs

```diff
@@ -483,11 +477,65 @@ pushCandidate (pullRequestId, pullRequest) state = do
 -- effects if it does not change the state.
 proceedUntilFixedPoint :: ProjectState -> Action ProjectState
 proceedUntilFixedPoint state = do
-  newState <- proceed state
+  newState <- provideFeedback state >>= proceed
```

Comment:
Ooh actually, after writing it this way, it is obvious what we should do! `provideFeedback` should just be part of `proceed`! It is only a cosmetic change, the behavior is the same, but it suddenly falls into place :smile: 

The purpose of `proceed` is to find the next thing it can make progress on, and to perform that action. So the first thing `proceed` should do is to check if any PRs need feedback, and if so, provide it, and unset the flag. Then once all feedback has been provided, `proceed` can no longer make progress there, so then it will continue with whatever it currently already does.